src/core/__tests__/util-test.js

```diff
@@ -182,6 +228,27 @@ jest.mock('../../data-services/public.js');
       expect(util.validateUnstructuredTimeseriesData({data: concatenatedTasmaxTimeseries})).toEqual({data: concatenatedTasmaxTimeseries});
     });
   });
+  
+  describe('validateWatershedData', function () {
+    it('rejects Workzeug error messages', function () {
+      var func = function () {util.validateUnstructuredTimeseriesData({data:
```

Comment:
`const`, please? Also could do `const func = () => { ... }`; briefer code, seems to be preferred in a lot of JS work lately.